cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1684,9 +1684,10 @@
 
   <property>
     <name>master.service.memory.mb</name>
-    <value>512</value>
+    <value>1024</value>
```

Comment:
Why need this change?

Reply:
this is per the jira: https://issues.cask.co/browse/CDAP-12767, and to keep configs consistent across CM/Ambari/Packages.  I believe it was in response to seeing the messaging container fail to startup.  We can discuss